charts/ingress-nginx/values.yaml

```diff
@@ -12,164 +12,159 @@ controller:
   dnsPolicy: ClusterFirst
 
-  # Bare-metal considerations via the host network https://kubernetes.github.io/ingress-nginx/deploy/baremetal/#via-the-host-network
+  # -- Bare-metal considerations via the host network https://kubernetes.github.io/ingress-nginx/deploy/baremetal/#via-the-host-network
   # Ingress status was blank because there is no Service exposing the NGINX Ingress controller in a configuration using the host network, the default --publish-service flag used in standard cloud setups does not apply
   reportNodeInternalIp: false
 
-  # Process Ingress objects without ingressClass annotation/ingressClassName field
+  # -- Process Ingress objects without ingressClass annotation/ingressClassName field
   # Overrides value for --watch-ingress-without-class flag of the controller binary
   # Defaults to false
   watchIngressWithoutClass: false
 
-  # Process IngressClass per name (additionally as per spec.controller)
+  # -- Process IngressClass per name (additionally as per spec.controller)
   ingressClassByName: false
 
-  # This configuration defines if Ingress Controller should allow users to set
+  # -- This configuration defines if Ingress Controller should allow users to set
   # their own *-snippet annotations, otherwise this is forbidden / dropped
   # when users add those annotations.
   # Global snippets in ConfigMap are still respected
   allowSnippetAnnotations: true
 
-  # Required for use with CNI based kubernetes installations (such as ones set up by kubeadm),
+  # -- Required for use with CNI based kubernetes installations (such as ones set up by kubeadm),
   # since CNI and hostport don't mix yet. Can be deprecated once https://github.com/kubernetes/kubernetes/issues/23920
   # is merged
   hostNetwork: false
 
   ## Use host ports 80 and 443
   ## Disabled by default
-  ##
   hostPort:
+    # -- Enable 'hostPort' or not
     enabled: false
     ports:
+      # -- 'hostPort' http port
       http: 80
+      # -- 'hostPort' https port
       https: 443
 
-  ## Election ID to use for status update
-  ##
+  # -- Election ID to use for status update
   electionID: ingress-controller-leader
 
-  # This section refers to the creation of the IngressClass resource
-  # IngressClass resources are supported since k8s >= 1.18 and required since k8s >= 1.19
+  ## This section refers to the creation of the IngressClass resource
+  ## IngressClass resources are supported since k8s >= 1.18 and required since k8s >= 1.19
   ingressClassResource:
     name: nginx
     enabled: true
     default: false
     controllerValue: "k8s.io/ingress-nginx"
 
-    # Parameters is a link to a custom resource containing additional
+    # -- Parameters is a link to a custom resource containing additional
     # configuration for the controller. This is optional if the controller
     # does not require extra parameters.
     parameters: {}
 
-  # labels to add to the pod container metadata
+  # -- Labels to add to the pod container metadata
```

Comment:
What is the result of this modification?

Reply:
Results in the following line in the generated `README.md` file => https://github.com/sc250024/ingress-nginx/blob/feat-NginxHelmChartDocs/charts/ingress-nginx/README.md?plain=1#L363

Reply:
ok